### PR TITLE
[branch: cnsmap] Adding Code Review, Availability, Development of Tests sections

### PR DESCRIPTION
**Pull request by dcomas**
Updates with some links the 3 sections. Can you please confirm the relative paths in Development of Test work well as I can't find the latest version of the Vercel app.

Thanks

### CODE REVIEW

File: content/code-review.md [hunk 1]
```diff
@@ -9,10 +9,14 @@ consequences. To mitigate the risk of unintended consequences, teams are encoura
...
+- [Review Board](https://www.reviewboard.org/)
+- [Kallithea](https://kallithea-scm.org/repos/kallithea)
 
 ## Examples
-- TODO
-
-## Links
-- TODO
+- Code review tools help ensure that ownership over code quality and security is shared.
+- Human error is a natural part of the development process, and code review tools help ensure that those errors are addressed swiftly.
```
Comment (ashutosh-narkar) [inline content/code-review.md:22]: These are good examples of why code review tools are useful. What would be better if we gave specific examples of the actual usage of these tools which users can adopt.

File: content/development-of-tests.md [hunk 2]
```diff
@@ -16,10 +16,16 @@ continuous integration (CI) feedback loops. System test suites should be made av
...
 
 ## Projects
-- TODO
-
-## Examples
-- TODO
+- [Static Analysis Testing](../static-analysis-and-security-testing.md)
+- [Dynamic Analysis Testing](../dynamic-analysis.md)
+- [Container Manifest Testing](../container-application-manifest.md)
+- [Threat Modelling](../threat-modelling.md)
```
Comment (lumjjb) [inline content/development-of-tests.md:22]: Hmm, i'm a little confused at this section. 

My impression for projects for this section were the TDD / unit testing frameworks, etc. which target more of test authoring by humans, which I think is the  intent of the section.

**Comment by TheFoxAtWork** [comment]
@dcomas could you address @lumjjb & @ashutosh-narkar 's comments?

**Comment by dcomas** [comment]
> @dcomas could you address @lumjjb & @ashutosh-narkar 's comments?

Yes I will do ASAP

File: content/development-of-tests.md (hunk 2 above)
Comment (dcomas) [reply content/development-of-tests.md:22]: As discussed in the catch up session, I will remove the static links and include more information regarding those items.

File: content/code-review.md (hunk 1 above)
Comment (dcomas) [reply content/code-review.md:22]: Yes, makes sense. I will add more specific examples, thanks!

**Review by lumjjb — APPROVED** [review]
lgtm

**Pull request merged**